src/avr910.c

```diff
@@ -326,12 +326,20 @@ static int avr910_parseextparms(const PROGRAMMER *pgm, const LISTID extparms) {
 
       continue;
     }
-    if (strncmp(extended_param, "no_blockmode", strlen("no_blockmode")) == 0) {
+    if (str_eq(extended_param, "no_blockmode")) {
```

Comment:
Cool - this  differs from what's been there before but the new version is actually what should be there.